build/configs/artik053/minimal/defconfig

```diff
@@ -768,6 +767,12 @@ CONFIG_EXAMPLES_HELLO=y
 # CONFIG_SYSTEM_FOTA_HAL is not set
 # CONFIG_SYSTEM_I2CTOOL is not set
 # CONFIG_SYSTEM_INIFILE is not set
+CONFIG_SYSTEM_PREAPP_INIT=y
```

Comment:
You said "his is not always needed". But in minimal, it is enabled. What is right?